src/Home.js

```diff
@@ -0,0 +1,11 @@
+import BikeSVG from "./BikeSVG"
+
+function Home() {
+    return (
+        <div className="mainContent">
+            <BikeSVG className="bikeSVG" />
+        </div>
```

Comment:
It might be a good idea to provide a div that provides context as to what your app does, and how to use it

Reply:
Added describing DIV